src/main/java/com/blamejared/compat/forestry/BaseAddForestry.java

```diff
@@ -0,0 +1,21 @@
+package com.blamejared.compat.forestry;
```

Comment:
This stuff should be in MTLib.

Reply:
This class is exclusively working for forestry. Why should anyone want to add something to forestry if it is already in modtweaker?